app/src/main/java/com/example/placefinder/service/ParkService.kt

```diff
@@ -8,8 +8,12 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
-class ParkService {
-    suspend fun fetchParks(): List<Park>? {
+interface IParkService {
```

Comment:
I have refactored an Interface for the ParkService class. This interface will hold functions that can be overridden once a class implements from that Interface. Please refer to this documentation on how to implement interfaces/the benefits of them: https://kotlinlang.org/docs/interfaces.html

Reply:
For the Dependency Inversion and Liskov Substitution SOLID principles, using interfaces is a good idea.